naughty/fedora-38/5874-lvconvert-hangs-in-kernel

```diff
@@ -0,0 +1,6 @@
+[<0>] md_stop+*
```

Comment:
Fedora 38 as well?

Reply:
That's more proactive. If it goes into F39, it's likely to go into F38 as well.